developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -80,7 +80,7 @@ Either of these methods will expose a [REST API service](../apis/fullnode-rest-a
 ### SDKs and tools
 
 Aptos currently provides three SDKs:
-1. [Typescript](../sdks/ts-sdk-v2/index.md)
+1. [Typescript](../sdks/new-ts-sdk/index.md)
```

Comment:
The path change will be confusing, did we link in the medium?

Reply:
yes, will change it there once this is in. I think we are still in the timeframe to change paths